README.md

```diff
@@ -228,14 +232,14 @@ After an upgrade of the buildpack or agent, you must clear your build cache and
 [5]: https://github.com/lstoll/heroku-buildpack-monorepo
 [6]: https://app.datadoghq.com/account/settings#api
 [7]: https://github.com/DataDog/heroku-buildpack-datadog/releases
-[8]: https://docs.datadoghq.com/tagging
```

Comment:
tagging guide link seems to have disappeared?

Reply:
yup it's no longer in the main README, the list of link is now up to date :)